cibuildwheel/__main__.py

```diff
@@ -244,6 +244,16 @@ def _compute_platform(args: CommandLineArguments) -> PlatformName:
     return _compute_platform_ci()
 
 
+def get_platform_interface(platform: PlatformName) -> PlatformInterface:
+    if platform == "linux":  # noqa: SIM116
+        return cibuildwheel.linux.interface
+    elif platform == "windows":
+        return cibuildwheel.windows.interface
+    elif platform == "macos":
```

Comment:
```suggestion
    if platform == "windows":
        return cibuildwheel.windows.interface
    if platform == "macos":
```

Given the final `assert_never` is unindented (which I prefer), these should be if's, not elif's.

(if this is still valid after trying @joerick's suggestion)